styles/styles.css

```diff
@@ -244,7 +250,13 @@ main img {
 
 /* Sections */
 main .section {
-  padding: 32px;
+  padding: 15px;
```

Comment:
can this be 16 instead of 15 (16 being half of 32 and also I have a weird thing with even numbers)